spec/quotas/quotas.mdown

```diff
@@ -0,0 +1,147 @@
+* `account`: Applies for this account
+* `domain`: All users of this domain share this part of the quota
+* `global`: All users of this mail server share this part of the quota
+
+## The Quota Object
+
+The quota object MUST contain the following fields:
+
+* **id**: `Id` The unique identifier for this object. It should respect the JMAP ID datatype defined in section 1.2 of [@!RFC8620]
+
+* **type**: `String` The type of quota. It should either be "messageCount" or "messageStorageSize". There is two ways we can define a quota: either on a number of emails, or in the amount of bytes.
+
+* **used**: "UnsignedInt" The current usage of the mailbox. Computation of this value is handled by the server.
+
+* **usedScope**: `Scope` The `Scope` of the current mailbox usage.
+
+* **limit**: "UnsignedInt" The limit set by this quota object to the current mailbox.
+
+* **limitScope**: `Scope` The `Scope` of the mailbox limit.
+
+The quota object MAY contain the following field:
+
+* **description**: `String|null` Arbitrary free, human readable, description of this quota. Might be used to explain where the limit comes from and explain the entities this quota applies to.
+
+## Example
+
+    {
+      "id": "2a06df0d-9865-4e74-a92f-74dcc814270e",
+      "type": "MESSAGE",
+      "used": 1056,
+      "usedScope": "account",
+      "limit": 2000,
+      "limitScope": "domain",
+      "description": "Personal account usage, bounded by a domain limit"
+    }
+
+## Quota/get
+
+Standard “/get” method as described in [@!RFC8620] section 5.1. The ids argument may be `null` to fetch all at once.
+
+### Example
+
+Request fetching all quotas related to an account :
+
+    [[ "Quota/get", {
+      "accountId": "u33084183",
+      "ids": null
+    }, "0" ]]
+
+With response :
+
+    [[ "Quota/get", {
+      "accountId": "u33084183",
+      "state": "78540",
+      "list": [{
+      "id": "2a06df0d-9865-4e74-a92f-74dcc814270e",
+      "type": "MESSAGE",
+      "used": 1056,
+      "usedScope": "account",
+      "limit": 2000,
+      "limitScope": "domain",
+      "description": "Personal account usage, bounded by a domain limit"
+      }, {
+        "id": "3b06df0e-3761-4s74-a92f-74dcc963501x",
+        "type": "STORAGE",
+        ...
+      }, ...],
+      "notFound": []
+    }, "0" ]]
+
+
+## Quota/changes
+
+Standard “/changes” method as described in [@!RFC8620] section 5.2.
+
+### Example
+
+Request:
+
+    [[ "Quota/changes", {
+      "accountId": "u33084183",
+      "sinceState": "10824",
+      "maxChanges": 20
+    }, "0" ]
+
+Response:
+
+    [[ "Quota/changes", {
+      "accountId": "u33084183",
+      "oldState": "10824",
+      "newState": "10826",
+      "hasMoreChanges": false,
+      "created": [],
+      "updated": ["2a06df0d-9865-4e74-a92f-74dcc814270e"],
+      "destroyed": []
+    }, "0" ],
```

Comment:
I think this lline is not in the right format, missing a bracket or something...